pandas/core/strings/accessor.py

```diff
@@ -1290,8 +1290,8 @@ def contains(
         1    False
         2    False
         3    False
-        4      NaN
-        dtype: object
+        4    False
+        dtype: bool
 
         Specifying `na` to be `False` instead of `NaN` replaces NaN values
```

Comment:
I realized that this example is now also a bit outdated (we could still show it when starting with object dtype, or show it with filling with another value), but that is also fine to keep for another PR

Reply:
-> https://github.com/pandas-dev/pandas/pull/61960